app/models/delete_queue_job.rb

```diff
@@ -22,6 +22,9 @@ def perform
       file_type = object.file_type
       study = object.study
 
+      # remove expression_file_info, if present to avoid validation issues later
```

Comment:
Slightly clearer:

```suggestion
      # remove any expression_file_info to avoid validation issues later
```

Reply:
do we need these lines?  I would just as soon rewrite the update below to use `save(validate: false)` so that we don't have to add corner cases up here every time a model validation changes

Reply:
You are correct in that we don't explicitly need them, but given that we were already having problems with the documents not saving correctly, and I'm _never_ going to feel comfortable with not enforcing validations on document save, I made the call to change it.  We knew the problem was the invalid `expression_file_info` document, and it seems more "correct" in my brain to address that directly.

Also, regarding future validation changes - we need to do a better job of migrating data and writing tests surrounding it.  I would rather have more tests/coverage around that then expedient solutions like disabling validations.